sdcm/cluster.py

```diff
@@ -2600,6 +2600,14 @@ def node_setup(self, node, verbose=False, db_node_address=None, **kwargs):
         node.remoter.run("source $HOME/.bashrc")
         node.remoter.run("go get github.com/scylladb/scylla-bench")
 
+        self.install_gemini_tool(node)
+
+    def install_gemini_tool(self, node):
+        result = node.remoter.run('test -e ~/gemini', ignore_status=True)
```

Comment:
if the version that is installed differs from the wanted version?

Reply:
1. validate version of installed and wanted, and if they differ, reinstall
or 
2. remove the installed version and install wanted